.github/workflows/build.yml

```diff
@@ -19,7 +21,25 @@ jobs:
         with:
           nix_path: nixpkgs=./nixpkgs.nix
 
-      - run: echo "build --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs" > .bazelrc.local
+      - name: Configure
+        run: |
+          mkdir -p ~/repo-cache ~/disk-cache
+          echo "build --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs" > .bazelrc.local
+
+      - name: Mount Bazel cache
+        uses: actions/cache/restore@v3
+        if: github.ref != 'refs/heads/master'
+        id: restore-cache
+        with:
+          path: |
+            ~/repo-cache
+            ~/disk-cache
+          key: repo-cache-${{ runner.os }}-nixpkgs-${{ env.cache-version }}-${{ github.run_id }}-${{ github.run_attempt }}
```

Comment:
`cache-version` needs to be set or this will be undefined.

Reply:
Oh misse that, I thought env.cache_version value in injected by githubaction.
I should have set cache_verison in `env` like this
`env:
  cache_version: ${{ github.sha }}`
  
  but any way cache key and restore key is getting to big so I am removing it.